AdminUi/apps/admin_ui/lib/home/home.dart

```diff
@@ -1,6 +1,7 @@
 export 'clients_overview/clients_overview.dart';
 export 'identities_overview/deletion_process_audit_log_details/deletion_process_audit_log_details.dart';
 export 'identities_overview/identities_overview.dart';
+export 'identity_details/deletion_process_table/deletion_process_details/deletion_process_details.dart';
```

Comment:
why are the deletion_process_details located in the deletion_process_table? I'd move this next to the deletion_process_table instead